mmcv/runner/base_module.py

```diff
@@ -38,12 +63,38 @@ def is_init(self):
 
     def init_weights(self):
         """Initialize the weights."""
-        from ..cnn import initialize
 
+        # judge if it is Topmost module
+        top_most_module = len(self.params_init_info) == 0
```

Comment:
- `top_most_module` is a noun, which is not suitable for a boolean value. `is_top_level_module` should be better